src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -1,33 +1,67 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Class implementing default stack functionality
+ */
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	private static final int EMPTY_STACK_VALUE = -1;
+	private static final int STACK_CAPACITY = 12;
 
-	public int total = -1;
+	private final int[] numbers = new int[STACK_CAPACITY];
 
-	public void countIn(int in) {
+	/**
+	 * Gets total amount of numbers placed on stack minus 1
+	 * @return total amount of numbers placed on stack minus 1
+	 */
+	public int getTotal() {

```

Comment:
field used only internally - should not have a getter

Reply:
There is an instruction to generate getter for total. 3.1.7